tests/integer_float_combination/integer_float_combination.mdl

```diff
@@ -0,0 +1,97 @@
+{UTF-8}
+Integer Integral= INTEG (
+	Value,
+		0)
+	~	
+	~		|
+
+Value=
+	1 + STEP(1,10)
+	~	
+	~		|
+
+Float Smooth=
+	SMOOTH N(input, smoothing time, input , order )
```

Comment:
Is it possible to write this test without needing the smooth? Possibly not - I haven't gone into detail on the issue. 

Ideally each test here should do basically one thing, so that we can isolate the problems, and this would also fail if there was an issue with SMOOTH, (or STEP). What do you think?

Reply:
I tried to run a test with integer/float integrals but it passes, I believe the problem arises when you are dealing with multi state Functions. I dont know if there is any other way to replicate this issue.
But maybe we can leave this test out and stick to the smooth_and_stock test  as it replicates the error. JamesPHoughton/pysd#149

Reply:
That sounds good. I'll let you update this branch, and then I'll merge the PR to bring in the TREND test